cob/project.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, root='.'):
         self.tests = os.path.abspath(root) + '/tests'
         self._static = {}
         self.tst_cfg_dir = self._tst_cfg_dir
+        self.ocf_dir = self._ocf_dir
```

Comment:
What is ocf_dir?

Reply:
This is a very confusing behavior - taking the result of a property and putting it in an attribute with a very similar name...

Reply:
this is indeed confusing and not needed. i fixed it.

Reply:
In my last commit i also removed the _tst_cfg_dir which had the exact same issue.
in both cases i just left the properties.